files/build_templates/sonic_debian_extension.j2

```diff
@@ -157,6 +157,10 @@ sudo sed -i -e '/^passwd/s/ tacplus//' $FILESYSTEM_ROOT/etc/nsswitch.conf
 # Copy crontabs
 sudo cp -f $IMAGE_CONFIGS/cron.d/* $FILESYSTEM_ROOT/etc/cron.d/
 
+## copy database instances checking script and config
+sudo cp $IMAGE_CONFIGS/database/ping_pong_single_db_inst.sh $FILESYSTEM_ROOT/usr/local/bin/
+sudo cp $IMAGE_CONFIGS/database/ping_pong_db_insts $FILESYSTEM_ROOT/usr/local/bin/
```

Comment:
suggest to remove these from base image, when base image needs to ping the database, it can use docker to get into the database docker and run ping.

Reply:
moved ping script into database docker